packages/common/src/hooks/useOpenInNewTab/useOpenInNewTab.ts

```diff
@@ -0,0 +1,15 @@
+import { useNavigate } from 'react-router';
+
+export const useOpenInNewTab = () => {
+  const navigate = useNavigate();
+  const { origin } = window.location;
```

Comment:
Could change this to not use the origin.. but then the `else` case shouldn't use `useNavigate` either..

Reply:
oops with some testing I realised I shouldn't be appending the origin like I was!